pkg/crd/kubermatic/v1/datacenter.go

```diff
@@ -206,6 +206,10 @@ type DatacenterSpecOpenstack struct {
 	// See https://kubernetes.io/docs/concepts/cluster-administration/cloud-providers/#load-balancer
 	// This setting defaults to true.
 	ManageSecurityGroups *bool `json:"manage_security_groups"`
+	// Optional: Gets mapped to the "use-octavia" setting in the cloud config.
+	// use-octavia is enabled by default in CCM since v1.17.0, and disabled by
+	// default with the in-tree cloud provider. This setting defaults to true.
```

Comment:
Looks better, but I would remove `This setting defaults to true.` if you don't mind as AFIK no default is given.